screener.config.js

```diff
@@ -8,11 +8,17 @@ module.exports = {
   baseBranch: "master",
   browsers: [
     {
-      browserName: "chrome"
+      browserName: "chrome",
+      version: "87.0"
```

Comment:
don't we want 89 or greater?

Reply:
We do, but 87 is currently the highest supported Chrome version according to: https://docs.saucelabs.com/visual/component-testing/integrations/sauce-labs/#supported-browsers

Reply:
Also, they don't have a `latest` option either, but they did submit that idea as product feedback.

Reply:
Didn't they say that doc was outdated in the email?

Reply:
>Please disregard the supported browsers section, as this appears to be outdated. I have reached out to our documentation team about this discrepancy.

Reply:
Wow, I totally overlooked that.  I reached out to them to ask them what is going on, so hopefully they can provide me the proper config.

Reply:
Can you change it to 89?

Reply:
Once we get it working on 87 we can bump it to see if it works on 89. Right now we just need to get it all set up correctly.